src/openms_gui/source/VISUAL/DIALOGS/SwathTabWidget.cpp

```diff
@@ -62,19 +84,190 @@ namespace OpenMS
         // call once to update py_pyprophet canvas 
         // alternative: load latest data from .ini and set py_selector (will update py_pyprophet via above signal/slot)
         py_pyprophet->validate(py_selector->getLastPython().toQString());
+
+        ui->input_tr->setFileFormatFilter("Transition sqLite file (*.pqp)");
+        ui->input_iRT->setFileFormatFilter("Transition sqLite file (*.pqp)");
+        ui->out_dir->setDirectory(getDefaultOutDir());
+
+        // create a default config from OpenSwathWorkflow
+        String tmp_file = File::getTemporaryFile();
+        QProcess qp;
+        qp.start(getOSWExe().toQString(), QStringList() << "-write_ini" << tmp_file.toQString());
+        qp.waitForFinished();
+        
+        ParamXMLFile().load(tmp_file, swath_param_);
+        swath_param_ = swath_param_.copy("OpenSwathWorkflow:1:", true);
+        // parameters to show:
+        StringList extract = {"mz_extraction_window", "rt_extraction_window", "threads"};
+        
+        for (const auto& name : extract) swath_param_wizard_.setValue(name, ""); // create a dummy param, just so we can use ::copySubset
+        swath_param_wizard_ = swath_param_.copySubset(swath_param_wizard_);
+                
+        ui->list_editor->load(swath_param_wizard_);
+
+        // keep the group of input widgets in sync with respect to their current-working-dir, when browsing for new files
+        connect(ui->input_mzMLs, &InputFileList::updatedCWD, this, &SwathTabWidget::broadcastNewCWD_);
+        connect(ui->input_iRT, &InputFile::updatedCWD, this, &SwathTabWidget::broadcastNewCWD_);
+        connect(ui->input_tr, &InputFile::updatedCWD, this, &SwathTabWidget::broadcastNewCWD_);
+        connect(ui->input_swath_windows, &InputFile::updatedCWD, this, &SwathTabWidget::broadcastNewCWD_);
     }
 
     SwathTabWidget::~SwathTabWidget()
     {
         delete ui;
     }
 
-    void SwathTabWidget::on_pushButton_clicked()
+    String infileToOSW(const String& infile)
```

Comment:
needed as public member?

Reply:
no. Its only used in SwathWidget.cpp. No need to export this.